### ISSUE TITLE
Build issue-fresh clone, "make" fails

### ISSUE THREAD

**Issue by galbacarys** [GoogleContainerTools/kaniko#118]
I tried both cloning the repo manually into the right location and using go get, and I've also made sure to run `dep ensure`.

When I try to run `make`, I get the following output:
```
$ make
GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -ldflags '-extldflags "-static" -X .version=v0.1.0 ' -tags "containers_image_ostree_stub containers_image_openpgp exclude_graphdriver_devicemapper exclude_graphdriver_btrfs exclude_graphdriver_overlay" -o out/executor github.com/GoogleCloudPlatform/kaniko/cmd/executor
can't load package: package github.com/GoogleCloudPlatform/kaniko/cmd/executor: cannot find package "github.com/GoogleCloudPlatform/kaniko/cmd/executor" in any of:
	/usr/local/opt/go/libexec/src/github.com/GoogleCloudPlatform/kaniko/cmd/executor (from $GOROOT)
	/Users/galbacarys/go/src/github.com/GoogleCloudPlatform/kaniko/cmd/executor (from $GOPATH)
make: *** [out/executor] Error 1
```

Clearly, there was a migration from GoogleCloudPlatform to GoogleContainerTools somewhere along the line, so the repository needs to be updated. A quick search reveals at least 30 references to the old package path. I can move the repo for now to temporarily solve the problem, but for the sake of ease of installation this should be fixed.

**Comment by priyawadhwa** [comment]
Sorry about that, we just changed the org today! Just merged #114, so if you pull these changes it should work.

**Comment by galbacarys** [comment]
Thanks!
